package.json

```
@@ -0,0 +1,28 @@
+{
+  "name": "project-arm-be",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "start": "nodemon index.js"
```

Better to associate this script with a 'dev' environment, because in most production cases you want to start an application with the 'node' command in order to avoid memory consumption overhead.
```suggestion
    "start": "node index.js"
    "start:dev": "npx nodemon index.js"
```